src/text.tsx

```diff
@@ -42,6 +42,7 @@ export interface TextPresetProps {
     role?: string;
     'aria-level'?: number;
     dataAttributes?: DataAttributes;
+    forceMobileSizes?: boolean;
```

Comment:
used to force the same text sizes in mobile and desktop (as requested by DisplayCards)

Reply:
wouldnt be weird to have a DisplayCard and a DataCard in desktop with different Text sizes? given the fact that this cards are pretty similar

Reply:
that's a question for Design Core

Reply:
@yceballost ?

Reply:
@HectorSancho

Reply:
In the tests we did, the desktop presets grew excessively and the text at maximum broke the component or was disproportionate when they appeared in a carousel and compared to other components. 

We believe that it scales better and also works better visually by limiting the scaling of the text-presets in desktop resolutions.